client/signup/config/steps-pure.js

```diff
@@ -398,6 +398,36 @@ export function generateSteps( {
 			stepName: 'site-information',
 			providesDependencies: [ 'siteTitle', 'address', 'email', 'phone' ],
 		},
+
+		// Steps with preview
+		// These can be removed once we make the preview the default
```

Comment:
Coming back to this... There was something I was missing. After toying with the site mockup component for the site style step, I see the merit in using such new step definitions, permitting us to slot them in and out of flows as necessary, and also avoiding creating new flows merely for the preview. 

I've rebased the branch and would like to use/expand on it for the site style step if that's ok. 👍

Reply:
Me again. I've subsumed this work under 
#29863, since it was pertinent to the new flow needed for business-only-site-preview-with-style-step stuff :)